### DISCUSSION TITLE
Thanks for an Incredible Add-on

### DISCUSSION THREAD

**Discussion by paulearley** [hassio-addons/addon-zwavejs2mqtt#87] [General]
I just wanted to say thanks SO much to the development group for an incredible add-on. I am using zwavejs2mqtt (WebSocket) to control my zwave components in Home Assistant. This came out in its initial versions in an amazingly stable condition. It has made my management of my home zwave network effortless.

**Comment by frenck** [comment]
I've passed along the message with the dev team! Thanks!
